src/lib/transit/station/station-service.ts

```
@@ -0,0 +1,155 @@
+    const toIndex = lineStations.findIndex((s) => s.id === toStation.id);
+
+    if (fromIndex === -1 || toIndex === -1) return [];
+
+    return fromIndex <= toIndex
+      ? lineStations.slice(fromIndex, toIndex + 1)
+      : lineStations.slice(toIndex, fromIndex + 1).reverse();
+  }
+
+  findAccessibleStations(
+    location: Coordinates,
+    count: number = 3,
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)